pkg/provisioner/controller.go

```diff
@@ -200,28 +200,23 @@ func (c *obcController) processNextItemInQueue() bool {
 func (c *obcController) syncHandler(key string) error {
 
 	setLoggersWithRequest(key)
-	logD.Info("new Reconcile iteration")
+	logD.Info("reconciling claim")
 
 	obc, err := claimForKey(key, c.libClientset)
 	if err != nil {
-		return fmt.Errorf("request key %q: %v", key, err)
-	}
-	if obc == nil {
-		return fmt.Errorf("unexpected nil obc for request key %q", key)
+		if errors.IsNotFound(err) {
+			log.Info("OBC vanished, assuming it was deleted")
```

Comment:
how about adding the key value in this log msg?

Reply:
the obc key is encoded in the logger.  is there something more we should put in?

Reply:
No. I forgot about the key being in the logger.

Reply:
Is the note comment above still accurate?